libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/ui/RouteArrowComponent.kt

```diff
@@ -44,7 +44,7 @@ class RouteArrowComponent(
         super.onDetached(mapboxNavigation)
         mapboxMap.getStyle()?.also { style ->
             routeArrowView.render(style, routeArrowApi.clearArrows())
-            RouteArrowUtils.removeLayers(style)
+            RouteArrowUtils.removeLayersAndSources(style)
```

Comment:
This line was added, because `MapboxRouteLineView` didn't apply the new options then. 
Since this PR fixes that, the line can be removed.

Reply:
line removed